src/content/en/tools/lighthouse/audits/webp.md

```diff
@@ -22,15 +25,25 @@ Click **View Details** to see the potential savings of encoding your page's
 images with WebP rather than JPEG or PNG. To pass this audit, encode all of
 these images in WebP.
 
-Browser support is not universal for WebP. You'll need to serve a fallback PNG
-or JPEG image when WebP is not available. See [How can I detect browser
-support for WebP?][fallback] for an overview of fallback techniques.
+Browser support is not universal for WebP, but similar savings should be available in
+most major browsers in an alternative next-gen format. You'll need to serve a fallback PNG
+or JPEG image for other browser support. See [How can I detect browser
+support for WebP?][fallback] for an overview of fallback techniques and the table below for browser 
+support of image formats.
 
 [fallback]: /speed/webp/faq#how_can_i_detect_browser_support_for_webp
 
+| Browser | WebP | JPEG 2000 | JPEG XR | JPEG | PNG |
```

Comment:
It'd be better to link this to somewhere else... we're not gonna remember to maintain this